mkdocs.yml

```diff
@@ -15,15 +15,31 @@ theme:
 # Pages
 nav:
   - Welcome: "index.md"
-  - Programming: "programming.md"
-  - Open science: "open_science.md"
-  - Open data: "open_data.md"
-  - Statistics: "statistics.md"
-  - Analysis software:
+  - Before you start:
+    - About: "01_Before-you-start.md"
+    - Programming: "programming.md"
+  - Planning your study:
+      - About: "02_Planning-your-study.md"
+      - Reproducible research: "reproducible_neuroimaging_research.md"
+  - Getting data:
+    - About: "03_Getting-data.md"
+    - Open data: "open_data.md"
+  - Processing:
+    - About: "04_Processing.md"
+    - Computing: "computing.md"
     - MRI: "analysis_software_MRI.md"
     - MEEG: "analysis_software_MEEG.md"
-  - Machine learning and Deep learning: machine_and_deep_learning.md
-  - Computing: "computing.md"
-  - learning neuroimaging: "learning_neuroimaging.md"
-  - Neuroimaging resources: "neuroimaging_resources.md"
-  - Reproducible research: "reproducible_neuroimaging_research.md"
+    - Learning neuroimaging: "learning_neuroimaging.md"
+    - Neuroimaging resources: "neuroimaging_resources.md"
+  - Quality control:
```

Comment:
Maybe I should have realized earlier but maybe quality control should come after the analysis, no?

If we said that we want to have QC as a separate section maybe it should come at the very end (before `reporting`)`and then we create those internal links to it.

Reply:
Hm, personally it makes intuitive sense to me for QC to be where it is (after processing and before analysis), but if you think it should be after analysis I will defer to you! I will shuffle it to the end.

Reply:
if you don't mind it would make me happy. :-)